src/commands/snapshot.ts

```diff
@@ -56,20 +56,22 @@ export default class Snapshot extends PercyCommand {
 
     const {args, flags} = this.parse(Snapshot)
 
-    const isWindows = process.platform === 'win32'
-
     const snapshotDirectory = args.snapshotDirectory as string
     const port = flags.port as number
     const staticServerPort = port + 1
     const networkIdleTimeout = flags['network-idle-timeout'] as number
     const baseUrl = flags['base-url'] as string
-    const ignoreFilesRegex = flags['ignore-files'] as string
-    const snapshotFilesRegex = flags['snapshot-files'] as string
+    const rawIgnoreGlob = flags['ignore-files'] as string
+    const rawSnapshotGlob = flags['snapshot-files'] as string
+
+    const snapshotGlobs = rawSnapshotGlob.split(',')
+
+    const ignoreGlobs = rawIgnoreGlob ? rawIgnoreGlob.split(',') : []
```

Comment:
Split on an empty string results in an array of an empty string. Globby thinks an empty string means all files, and we def do not want to ignore all files by default. If the raw string is empty then the empty string is falsey and we pass an empty array to the static snapshot service, otherwise we pass an array of the ignore globs.